src/pages/SearchPage.jsx

```diff
@@ -127,7 +128,7 @@ export default function SearchPage() {
         </div>
       </div>
       <div id="detail">
-        {moviesList ? <Outlet context={[moviesList]} /> : ""}
+        {moviesList ? <Outlet context={[moviesList, isFetchingData]} /> : ""}
```

Comment:
Since the initial value is [] there is no condition where this condition would fail. It will always render outlet. Then what is the point of this condition check?